euphonic/cli/show_sampling.py

```diff
@@ -95,20 +95,17 @@ def main(params: Optional[list[str]] = None) -> None:
             marker='x',
         )
 
-    elif args.sampling == 'random-sphere':
-        ax.scatter(
-            *zips(*euphonic.sampling.random_sphere(args.npts)),
-            marker='x',
-        )
-
     elif args.sampling == 'recurrence-cube':
         ax.scatter(
             *zips(
                 *euphonic.sampling.recurrence_sequence(args.npts, order=3)),
             marker='x',
         )
 
-    else:
-        raise ValueError('Sampling type f{args.sampling} is not implemented.')
+    else:  # random-sphere
```

Comment:
Why remove the invalid check?

Reply:
Because the argument goes through an argparse `choices` constraint this code is unreachable

Reply:
I still personally prefer the explicit check rather than comment, but I'm happy.

Reply:
I do too, really. Maybe it can be tested by monkey-patching argparse... Or maybe we just accept that these things are untestable. Either way I don't like ending this with an `elif` and no `else`.